src/usr/local/www/diag_backup.php

```diff
@@ -233,18 +233,17 @@ function check_and_returnif_section_exists($section) {
 					/* read the file contents */
 					$data = file_get_contents($_FILES['conffile']['tmp_name']);
 					if (!$data) {
-						log_error(sprintf(gettext("Warning, could not read file %s"), $_FILES['conffile']['tmp_name']));
-						return 1;
-					}
-
-					if ($_POST['decrypt']) {
+						$input_errors[] = gettext("Warning, could not read file {$_FILES['conffile']['tmp_name']}");
+					} else if ($_POST['decrypt']) {
```

Comment:
This should be `elseif` ([ref](https://docs.netgate.com/pfsense/en/latest/development/developer-style-guide.html#general-rules))

Reply:
fixed